config/sso.go

```diff
@@ -18,10 +18,14 @@ type SSOConfig struct {
 	Scopes        []string        `json:"scopes,omitempty"`
 	SessionExpiry metav1.Duration `json:"sessionExpiry,omitempty"`
 	// customGroupClaimName will override the groups claim name
-	CustomGroupClaimName string   `json:"customGroupClaimName,omitempty"`
-	UserInfoPath         string   `json:"userInfoPath,omitempty"`
-	InsecureSkipVerify   bool     `json:"insecureSkipVerify,omitempty"`
-	FilterGroupsRegex    []string `json:"filterGroupsRegex,omitempty"`
+	CustomGroupClaimName string `json:"customGroupClaimName,omitempty"`
+	UserInfoPath         string `json:"userInfoPath,omitempty"`
+	InsecureSkipVerify   bool   `json:"insecureSkipVerify,omitempty"`
+  FilterGroupsRegex    []string `json:"filterGroupsRegex,omitempty"`
+	// Add support for providing PrivateKey, ClientID and ClientSecret via env var
```

Comment:
```suggestion
	// support providing PrivateKey, ClientID and ClientSecret via env var
```

-  "_add_ support" would be outdated when this is merged, so just use "support"